source/utilities.tex

```diff
@@ -15367,15 +15367,15 @@
 %% FIXME: their Unicode characters are not available in our font (Latin Modern).
 \begin{example}
 \begin{codeblock}
-string s0 = format("[{}]", "h\tllo");                   // \tcode{s0} has value: \tcode{[h    llo]}
-string s1 = format("[{:?}]", "h\tllo");                 // \tcode{s1} has value: \tcode{["h\textbackslash tllo"]}
-string s3 = format("[{:?}] [{:?}]", '\'', '"');         // \tcode{s3} has value: \tcode{['\textbackslash '', '"']}
+string s0 = format("[{}]", "h\tllo");                  // \tcode{s0} has value: \tcode{[h    llo]}
+string s1 = format("[{:?}]", "h\tllo");                // \tcode{s1} has value: \tcode{["h\textbackslash tllo"]}
+string s3 = format("[{:?}, {:?}]", '\'', '"');         // \tcode{s3} has value: \tcode{['\textbackslash '', '"']}
 
 // The following examples assume use of the UTF-8 encoding
 string s4 = format("[{:?}]", string("\0 \n \t \x02 \x1b", 9));
-                                                        // \tcode{s4} has value: \tcode{[\textbackslash u\{0\} \textbackslash n \textbackslash t \textbackslash u\{2\} \textbackslash u\{1b\}]}
-string s5 = format("[{:?}]", "\xc3\x28");               // invalid UTF-8
-                                                        // \tcode{s5} has value: \tcode{["\textbackslash x\{c3\}\textbackslash x\{28\}"]}
+                                                       // \tcode{s4} has value: \tcode{["\textbackslash u\{0\} \textbackslash n \textbackslash t \textbackslash u\{2\} \textbackslash u\{1b\}"]}
```

Comment:
Adding the quotes caused the build to fail due to
"Overfull \hbox (3.15547pt too wide) in paragraph at lines 15376--15377"
removing one space fixes the issue, an alternative would be to split the result of `s4` in two lines.

Reply:
> removing one space fixes the issue

The checks suggest otherwise.

Try the latter.

Reply:
> > removing one space fixes the issue
> 
> The checks suggest otherwise.

It fixes that issue, but brings up another. That's why splitting the comment should actually work.

Reply:
Removing the space gives another error, it seems the comments must be aligned on a multiple of 4.
That is detected by a different script.
But I'll take your advice and split the comment over two lines.

Reply:
That's right. Can you remove four spaces from all the comments maybe? If not we may need to present this a bit differently.

Reply:
4 spaces would work too. I just changed it to two lines and pushed that before I read your comment.
@tkoeppe what do you prefer the current two lines or 4 reducing the indention by 4?

Reply:
If the entire example can use the same column for comments (even if that's four less than before), that's perfectly fine.